analysed-packages/libsoup/version-2.74.2/libsoup-2.74.2-SPDX2TV.spdx

```diff
@@ -0,0 +1,5397 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_libsoup-2.74.2.tar.xz_1718000596.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of libsoup 2.74.2.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-06-10T06:23:17Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: libsoup-2.74.2.tar.xz
+PackageFileName: libsoup-2.74.2.tar.xz
+SPDXID: SPDXRef-upload243
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 0d7aa8bf390624bf410263ca06714c9dcf7f1003
+PackageChecksum: SHA1: 21177dcd3b04a10cacaa5b22bf233a4f2691eef6
+PackageChecksum: SHA256: f0a427656e5fe19e1df71c107e88dfa1b2e673c25c547b7823b6018b40d01159
+PackageChecksum: MD5: d8c5bc7f471c7462c29cc2ccebe3d9ca
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-LGPL-2.0-only
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
```

Comment:
Please add the content of the Authors file here too, since you have made it available in the disclosure document